cypress/fixtures/analysis.json

```diff
@@ -59,58 +52,182 @@
+                "category": "mandatory",
+                "source": ["None"],
+                "targets": ["eap", "eap8", "jakarta-ee", "jakarta-ee9+", "jws", "jws6+"],
+                "effort": 1,
+                "totalEffort": 750,
+                "incidents": 2,
+                "affectedFiles": 2,
+                "ruleSet": "eap8/eap7",
+                "rule": "javax-to-jakarta-dependencies-00002",
+                "labels": ["konveyor.io/source"]
             },
             {
-                "name": "taglibs.standard",
-                "language": "java",
-                "labels": []
+                "name": "The package 'javax' has been replaced by 'jakarta'.",
+                "category": "mandatory",
+                "source": ["None"],
+                "targets": ["eap", "eap8", "jakarta-ee", "jakarta-ee9+", "jws", "jws6+"],
+                "effort": 1,
+                "totalEffort": 750,
+                "incidents": 639,
+                "affectedFiles": 92,
+                "ruleSet": "eap8/eap7",
+                "rule": "javax-to-jakarta-import-00001",
+                "labels": ["konveyor.io/source"]
+            },
+            {
+                "name": "javax groupId has been replaced by jakarta.platform",
+                "category": "mandatory",
+                "source": ["None"],
+                "targets": ["eap", "eap8", "jakarta-ee", "jakarta-ee9+", "jws", "jws6+"],
+                "effort": 1,
+                "totalEffort": 750,
+                "incidents": 2,
+                "affectedFiles": 2,
+                "ruleSet": "eap8/eap7",
+                "rule": "javax-to-jakarta-dependencies-00006",
+                "labels": ["konveyor.io/source"]
+            },
+            {
+                "name": "javax javaee-api artifactId has been replaced by jakarta.platform jakarta.jakartaee-api",
+                "category": "mandatory",
+                "source": ["None"],
+                "targets": ["eap", "eap8", "jakarta-ee", "jakarta-ee9+", "jws", "jws6+"],
+                "effort": 1,
+                "totalEffort": 750,
+                "incidents": 2,
+                "affectedFiles": 2,
+                "ruleSet": "eap8/eap7",
+                "rule": "javax-to-jakarta-dependencies-00007",
+                "labels": ["konveyor.io/source"]
+            },
+            {
+                "name": "web.xml element references a javax-prefixed class name",
+                "category": "potential",
+                "source": ["None"],
+                "targets": ["eap", "eap8", "jakarta-ee", "jakarta-ee9+", "jws", "jws6+"],
+                "effort": 1,
+                "totalEffort": 750,
+                "incidents": 7,
+                "affectedFiles": 1,
+                "ruleSet": "eap8/eap7",
+                "rule": "javax-to-jakarta-servlet-00130",
+                "labels": ["konveyor.io/source"]
             }
         ]
     },
 
+    "source+dep_analysis_on_tackletestapp": {
```

Comment:
It is already there

Reply:
As we talked, it just moved to another part of the file, so it is OK